deploy/docker-compose/server-rvi-client.yml

```diff
@@ -0,0 +1,97 @@
+version: '2'
+services:
+  
+  mysql:
+    image: advancedtelematic/mariadb
+    ports:
+      - '3306:3306'
+    expose:
+      - '3306'
+    environment:
+      MYSQL_ROOT_PASSWORD: "sota-test"
+      MYSQL_USER: "sota"
+      MYSQL_PASSWORD: "s0ta"
+      MYSQL_DATABASES: "sota_resolver sota_resolver_test sota_core sota_core_test"
+
+
+  rvi_server:
+    image: advancedtelematic/rvi
+    ports:
+      - 8801:8801
+      - 8807:8807
+    depends_on:
+      - mysql
+    stdin_open: true
+    tty: true
+    command: server
+
+  resolver:
+    image: advancedtelematic/sota-resolver
+    ports:
+      - 8081:8081
+    expose:
+      - 8081
+    depends_on:
+      - mysql
+      - rvi_server
+      - core
+    environment:
+      HOST: 0.0.0.0
+      RESOLVER_DB_URL: "jdbc:mariadb://mysql:3306/sota_resolver"
+      RESOLVER_DB_MIGRATE: 'true'     
+
+  core:
+    image: advancedtelematic/sota-core
+    ports:
+      - 8080:8080
+    expose:
+      - 8080
+    depends_on:
+      - mysql
+      - rvi_server
```

Comment:
That would create a circular dependency. In docker-compose v.2, by default all the containers are on a network and can address each other by name. (you can explicitly define other topologies, but that's the default)